workflows/parsl_csa/workflow_preprocess.py

```diff
@@ -39,6 +39,7 @@
             address='127.0.0.1',
             label="htex_preprocess",
             cpu_affinity="alternating",
+            available_accelerators=params["available_accelerators"],
```

Comment:
@priyanka9991 I think only DeepCDR requires GPUs during preprocessing. For models that don't require GPUs for preprocessing, would removing `available_accelerators=params["available_accelerators"]` result in faster execution?

Reply:
Yes that is right. For example if 
available_accelerators=["0","1"]
workflow_preprocess.py will only start 2 processes.
But if we do not use available_accelerators, it will start multiple CPU processes depending on availability and it is much faster.